docs/CONTRIBUTING.md

```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 8. Under "Scopes", select "bot"
 9. Permissions. Enable only these permissions:
 
--   Manage Server
 -   Manage Roles
 -   Manage Channels
 -   Kick Members
 -   Ban Members
--   Manage Emojis
+-   Manage Emojis & Stickers
 -   Manage Webhooks
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
```

Comment:
Voice permissions are not needed. We can revert this change

Reply:
rolled back changes here ✅